common/src/main/java/com/wynntils/features/user/QuestInfoOverlayFeature.java

```diff
@@ -43,35 +43,68 @@ public QuestInfoOverlay() {
         }
 
-        TextRenderTask[] toRender = {
-            new TextRenderTask(
-                    "Tracked Quest Info:",
-                    TextRenderSetting.getWithHorizontalAlignment(
-                            this.getWidth(), CommonColors.GREEN, this.getRenderHorizontalAlignment())),
-            new TextRenderTask(
-                    "",
-                    TextRenderSetting.getWithHorizontalAlignment(
-                            this.getWidth(), CommonColors.ORANGE, this.getRenderHorizontalAlignment())),
-            new TextRenderTask(
-                    "",
-                    TextRenderSetting.getWithHorizontalAlignment(
-                            this.getWidth(), CommonColors.WHITE, this.getRenderHorizontalAlignment()))
-        };
+        List<TextRenderTask> toRender = List.of(
+                new TextRenderTask(
+                        "Tracked Quest Info:",
+                        TextRenderSetting.getWithHorizontalAlignment(
+                                this.getWidth(), CommonColors.GREEN, this.getRenderHorizontalAlignment())),
+                new TextRenderTask(
+                        "",
+                        TextRenderSetting.getWithHorizontalAlignment(
+                                this.getWidth(), CommonColors.ORANGE, this.getRenderHorizontalAlignment())),
+                new TextRenderTask(
+                        "",
+                        TextRenderSetting.getWithHorizontalAlignment(
+                                this.getWidth(), CommonColors.WHITE, this.getRenderHorizontalAlignment())));
 
-        @Override
-        protected void onConfigUpdate(ConfigHolder configHolder) {
-            recalculateRenderTasks();
-        }
+        List<TextRenderTask> toRenderPreview = List.of(
+                new TextRenderTask(
+                        "Tracked Quest Info:",
+                        TextRenderSetting.getWithHorizontalAlignment(
+                                this.getWidth(), CommonColors.GREEN, this.getRenderHorizontalAlignment())),
+                new TextRenderTask(
+                        "Test quest:",
+                        TextRenderSetting.getWithHorizontalAlignment(
+                                this.getWidth(), CommonColors.ORANGE, this.getRenderHorizontalAlignment())),
+                new TextRenderTask(
+                        "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer tempus purus in lacus pulvinar dictum. Quisque suscipit erat pellentesque egestas volutpat.",
+                        TextRenderSetting.getWithHorizontalAlignment(
+                                this.getWidth(), CommonColors.WHITE, this.getRenderHorizontalAlignment())));
 
         private void recalculateRenderTasks() {
-            toRender[0].setSetting(TextRenderSetting.getWithHorizontalAlignment(
-                    this.getWidth(), CommonColors.GREEN, this.getRenderHorizontalAlignment()));
+            toRender.get(0)
```

Comment:
I don't really want to drag this out further (just had a final look through before I was about to approve the whole thing), but I could not help realize that these recalculate functions might be made simpler and more similar by moving the GREEN, ORANGE and WHITE into a final static array, and looping over the toRender and color array. (In fact, they could be made so similar so as to make them a single method, which takes in the toRender or toRenderPreview arrays as argument. (That would almost get us to the point I were requesting before, with just changing the "input source".)